OneSignalSDK/onesignal/src/main/java/com/onesignal/outcomes/OSOutcomeTableProvider.java

```diff
@@ -0,0 +1,126 @@
+package com.onesignal.outcomes;
+
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteException;
+
+import com.onesignal.influence.model.OSInfluenceChannel;
+import com.onesignal.outcomes.OSOutcomesDbContract.CachedUniqueOutcomeTable;
+import com.onesignal.outcomes.OSOutcomesDbContract.OutcomeEventsTable;
+
+public class OSOutcomeTableProvider {
+
+    private static final String INTEGER_PRIMARY_KEY_TYPE = " INTEGER PRIMARY KEY";
+    private static final String TEXT_TYPE = " TEXT";
+    private static final String INT_TYPE = " INTEGER";
+    private static final String FLOAT_TYPE = " FLOAT";
+    private static final String TIMESTAMP_TYPE = " TIMESTAMP";
+    private static final String COMMA_SEP = ",";
```

Comment:
We aren't using this everywhere and its much longer text-wise. We should just remove this or be consistent and use it everywhere. 

We should properly just replace strings that are actual text. I think I added this `COMMA_SEP` a while ago, cant rem though